boot/bootutil/include/bootutil/crypto/ecdsa_p256.h

```diff
@@ -120,46 +136,42 @@ static inline int bootutil_ecdsa_p256_verify(bootutil_ecdsa_p256_context *ctx,
 typedef mbedtls_ecdsa_context bootutil_ecdsa_p256_context;
 static inline void bootutil_ecdsa_p256_init(bootutil_ecdsa_p256_context *ctx)
 {
-    mbedtls_ecdsa_init(ctx);
+	mbedtls_ecdsa_init(ctx);
 }
 
 static inline void bootutil_ecdsa_p256_drop(bootutil_ecdsa_p256_context *ctx)
 {
-    mbedtls_ecdsa_free(ctx);
+	mbedtls_ecdsa_free(ctx);
 }
 
 static inline int bootutil_ecdsa_p256_verify(bootutil_ecdsa_p256_context *ctx,
-                                             uint8_t *pk, size_t pk_len,
-                                             uint8_t *hash,
-                                             uint8_t *sig, size_t sig_len)
+					     uint8_t *pk, size_t pk_len,
+					     uint8_t *hash, uint8_t *sig,
+					     size_t sig_len)
 {
-    int rc;
+	int rc;
 
-    (void)sig;
-    (void)hash;
+	(void)sig;
+	(void)hash;
 
-    rc = mbedtls_ecp_group_load(&ctx->grp, MBEDTLS_ECP_DP_SECP256R1);
-    if (rc) {
-        return -1;
-    }
+	rc = mbedtls_ecp_group_load(&ctx->grp, MBEDTLS_ECP_DP_SECP256R1);
+	if (rc)
```

Comment:
Why were the {} removed? It's not generally a good idea, since it can lead to unexpected behavior if someone adds a statement.

Reply:
That caused a checkpatch warning again